proof/refine/ARM/Ipc_R.thy

```diff
@@ -2661,7 +2661,7 @@ lemma cteInsert_ct'[wp]:
 lemma maybeDonateSc_corres:
   "corres dc (tcb_at tcb_ptr and ntfn_at ntfn_ptr and invs and weak_valid_sched_action
               and valid_ready_qs and active_sc_valid_refills and valid_release_q
-              and current_time_bounded 1 and ex_nonz_cap_to tcb_ptr)
+              and current_time_bounded 2 and ex_nonz_cap_to tcb_ptr)
```

Comment:
Given that it was I who introduced that parameter to `current_time_bounded`, I have only myself to blame. But is it possible to replace this `2` with a non magic-number or remove the paramater altogether? Does the new strategy for dealing with current_time allow for the latter?

Reply:
We could certainly get rid of the number in `current_time_bounded`. We'd currently like `current_time_bounded 5` true after we call `getCurrentTime`. So we could define `current_time_bounded_const` as `5` (and even further abstract `getCurrentTime_buffer` to be `kernelWCET_ticks + current_time_bounded_const * MAX_PERIOD` if we'd like). The downside would be that many lemas would then be made weaker, because they require smaller constants like `1` or `2`. 

Any preference? This might be better put as a JIRA issue for now, and we can decide what'd we'd like later on when things are more settled

Reply:
JIRA issue sounds sensible.

Reply:
https://sel4.atlassian.net/browse/VER-1397